app/views/branding_groups/_form.html.erb

```diff
@@ -11,73 +11,83 @@
-    <div class="col-md-3">
-      <%= f.label :banner_image, 'Banner Image (suggested size: 1273x245)' %><br/>
-      <%= f.file_field :banner_image, class: 'btn btn-default fileinput-button' %>
-      <% if @branding_group.banner_image.present? %>
-        <p class="help-block">Current file: <%= link_to "<i class='fas fa-download'></i> #{@branding_group.banner_image_file_name} (#{ number_to_human_size @branding_group.banner_image_file_size})".html_safe,
-                                                        @branding_group.banner_image.url, download: @branding_group.banner_image_file_name,
-                                                        class: 'btn btn-xs btn-primary' %></p>
-      <% end %>
-    </div>
-    <div class="col-md-3">
-      <%= f.label :splash_image, 'Splash Image (suggested size: 383x89)' %><br/>
-      <%= f.file_field :splash_image, class: 'btn btn-default fileinput-button' %>
-      <% if @branding_group.splash_image.present? %>
-        <p class="help-block">Current file: <%= link_to "<i class='fas fa-download'></i> #{@branding_group.splash_image_file_name} (#{ number_to_human_size @branding_group.splash_image_file_size})".html_safe,
-                                                        @branding_group.splash_image.url, download: @branding_group.splash_image_file_name,
-                                                        class: 'btn btn-xs btn-primary' %></p>
-      <% end %>
-    </div>
-    <div class="col-md-3">
-      <%= f.label :footer_image, 'Footer Image (suggested size: 185x54)' %><br/>
-      <%= f.file_field :footer_image, class: 'btn btn-default fileinput-button' %>
-      <% if @branding_group.footer_image.present? %>
-        <p class="help-block">Current file: <%= link_to "<i class='fas fa-download'></i> #{@branding_group.footer_image_file_name} (#{ number_to_human_size @branding_group.footer_image_file_size})".html_safe,
-                                                        @branding_group.footer_image.url, download: @branding_group.footer_image_file_name,
-                                                        class: 'btn btn-xs btn-primary' %></p>
-      <% end %>
+  <div class="form-group form-check form-check-inline">
+    <%= f.label :public, 'Publish collection <span class="fas fa-question-circle" data-toggle="tooltip" data-placement="top" title="Select to have your collection listed in the Collections Directory. Visibility of individual studies within the collection is still controlled by study permissions set by the study owner."></span>'.html_safe, class: "form-check-label" %> &nbsp;
+    <%= f.check_box :public, class: "form-check-input" %>
+  </div>
+  <div class="form-group">
+    <%= f.label :user_id, 'Associated User' %><br />
+    <%= f.select :user_id, options_from_collection_for_select(User.all, :id, :email, @branding_group.new_record? ? nil : @branding_group.user_id),
+                 {include_blank: 'Please select a user account'}, class: 'form-control' %>
```

Comment:
We don't preface other defaults with pleasantries, so this seems worth omitting for brevity and consistency.

```suggestion
                 {include_blank: 'Select a user account'}, class: 'form-control' %>
```